dags/openshift_nightlies/util/slack_integration.py

```diff
@@ -1,17 +1,49 @@
 from airflow.hooks.base import BaseHook
 from airflow.providers.slack.operators.slack_webhook import SlackWebhookOperator
+import logging
+from os import environ
+
 SLACK_CONN_ID = 'slack'
+
+# Used to get the git user for the repo the dags live in. 
+def get_git_user():
+    git_repo = environ['GIT_REPO']
+    git_path = git_repo.split("https://github.com/")[1]
+    git_user = git_path.split('/')[0]
+    return git_user.lower()
+
```

Comment:
can this task just import the var_loader module and use the `get_git_user()` function from that instead of duplicating?

Reply:
@whitleykeith so the reason i added it here was coz importing it from var_loader was creating a circular import error, which was leading to dag import error.

Reply:
@whitleykeith so i was able to fix the circular error , by not importing executor in var_loader , it wasn't used in it and honestly there's no point/need to import executor.py in var_loader.py